src/rng/backends/curand/CMakeLists.txt

```diff
@@ -60,13 +60,14 @@ set(LIB_NAME onemkl_rng_curand)
 set(LIB_OBJ ${LIB_NAME}_obj)
 find_package(cuRAND REQUIRED)
 
-add_library(${LIB_NAME})
-add_library(${LIB_OBJ} OBJECT
-  philox4x32x10.cpp
+set(SOURCES philox4x32x10.cpp
   mrg32k3a.cpp
-  $<$<BOOL:${BUILD_SHARED_LIBS}>: mkl_rng_curand_wrappers.cpp>
+  $<$<BOOL:${BUILD_SHARED_LIBS}>: mkl_rng_curand_wrappers.cpp>)
```

Comment:
Extra `)`

Reply:
thanks for catching these, I forgot to build the cublas and curand backend apparently while testing.